php/commands/site.php

```diff
@@ -985,33 +987,63 @@ public function import($args, $assoc_args) {
    * [--site=<site>]
    * : Site to use
    *
-   * [--change-to-org=<org>]
-   * : Change the instrument to an Org by setting the id. ( must be admin )
+   * [--change-to=<UUID>]
```

Comment:
`change-to` seems awfully verbose. I'd prefer just using  `--set=<uuid>` (though maybe we have a better convention. it seems like we tend to use the name of the key, so it would be `--instrument=<uuid>`